src/routes/AuthRoute.ts

```diff
@@ -1,8 +1,9 @@
-import { login } from '../controllers/AuthController';
+import * as authController from '../controllers/AuthController';
 import { Router } from 'express';
 
 const authRouter = Router();
 
-authRouter.post('/signin', login);
+authRouter.post('/login', authController.login);
+authRouter.post('/check-login', authController.checkLogin);
```

Comment:
I am not the expert :sweat_smile:, I do not think it is a best practice to do a check-login endpoint, I think they do it as a class or a func and call it whenever they need to check authentication :eyes:

Reply:
We need it to be endpoint so we can check if user logged in so if he isn't we redirect it to login. This endpoint is mainly for front end. Their is another middleware to auth user and used only for protecting our endpoints.